tensorflow/lite/micro/kernels/unidirectional_sequence_lstm_test_config.cc

```diff
@@ -471,9 +471,9 @@ LstmIntegerTestConfig lstm_integer_peephole_config = {
     .output = lstm_output_peephole,
     .expected_output =
         (int8_t[kLstmNumBatchIntegerPeephole * kLstmSeqLengthIntegerPeephole *
-                kLstmNumOutputIntegerPeephole]){127, 127, -16, -21, 127, 127,
-                                                23, 127, 127, -128, 127, 127,
-                                                127, 127, 127, -128, 127, 127},
+                kLstmNumOutputIntegerPeephole]){127,127, -16, -21, 127, 127,
+                                                -128, 127, 127, -128, 127, 127,
+                                                127, 127, 105, -128, 127, 127},
```

Comment:
this is quite a large difference in the overall output compared to the previous implementation. Can you confirm that this is expected and not a bug?